cmd/launcher/options.go

```diff
@@ -298,15 +298,38 @@ func findOsquery() string {
 		osqBinaryName = osqBinaryName + ".exe"
 	}
 
-	if exPath, err := os.Executable(); err != nil {
-		maybeOsq := filepath.Join(filepath.Base(exPath), osqBinaryName)
+	var likelyDirectories []string
+
+	if exPath, err := os.Executable(); err == nil {
+		likelyDirectories = append(likelyDirectories, filepath.Dir(exPath))
+	}
+
+	// Places to check. We could conditionalize on GOOS, but it doesn't
+	// seem important.
+	likelyDirectories = append(
+		likelyDirectories,
+		"/usr/local/kolide/bin",
```

Comment:
do we want to add `/usr/local/kolide-k2/bin` here?

Reply:
Kinda blurs the line between the OSS side, and our normal SaaS usage, but I think it's reasonable. Good call.